superset/models/core.py

```diff
@@ -350,6 +350,19 @@ def set_sqlalchemy_uri(self, uri: str) -> None:
         conn = conn.set(password=PASSWORD_MASK if conn.password else None)
         self.sqlalchemy_uri = str(conn)  # hides the password
 
+    def mask_password_in_ssh_tunnel(self, ssh_tunnel: "SSHTunnel") -> None:
+        if ssh_tunnel.password is not None and ssh_tunnel.password != PASSWORD_MASK:
+            # Mask the real password with our password mask
+            self.ssh_tunnel["password"] = PASSWORD_MASK  # pylint: disable=no-member
+        if (
+            ssh_tunnel.private_key_password is not None
+            and ssh_tunnel.private_key_password != PASSWORD_MASK
+        ):
+            # Mask the real password with our password mask
+            self.ssh_tunnel[  # pylint: disable=no-member
+                "private_key_password"
+            ] = PASSWORD_MASK
```

Comment:
this isn't overwriting the actual value of the password before saving right?

Reply:
Yeah, probably it's best to just getting rid of it in the response. Thanks!